ArjunaJTS/interop/pom.xml

```diff
@@ -51,6 +52,28 @@
     </repository>
   </repositories>
 
+  <profiles>
+    <profile>
+      <id>include-for-JDK11-above-versions</id>
+      <activation>
+        <jdk>[11,)</jdk>
+      </activation>
+      <modules>
+        <module>glassfish</module>
+      </modules>
+    </profile>
+    <profile>
+      <id>skip-for-JDK9-all-versions</id>
```

Comment:
do you think you may add an comment why it's needed to skip for JDK9 but not for JDK11, for the further notice

Reply:
Because some modules in Narayana are not working with JDK9, therefore I had to skip it. Currently it is working with JDK8 and JDK11. So maybe once Narayana executes with JDK9 properly, we could remove JDK9 skip profile from here.